cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -0,0 +1,193 @@
+:_mod-docs-content-type: ASSEMBLY
+[id=“cloud-experts-rosa-hcp-activation-and-account-linking-tutorial”]
+= Tutorial: ROSA with HCP activation and account linking
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-rosa-hcp-activation-and-account-linking-tutorial
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2024-02-08
+//---
+//date: '2024-02-08'
+//title: Jiri Fiala
+//authors:
+//  - Jiri Fiala
+//---
+
+This tutorial describes the process for activating {hcp-title-first} and linking to an AWS account, before deploying the first cluster.
+
+[IMPORTANT]
+====
+If you have received a private offer for the product, make sure to proceed according to the instructions provided with the private offer before following this tutorial. The private offer is designed either for a case when the product is already activated, which replaces an active subscription, or for first time activations.
+====
+
+== Subscription enablement and AWS account setup
+
+. Activate the {hcp-title} product at the AWS console page by clicking the *Get started* button:
++
+image::rosa-get-started.png[]
++
+If you have activated ROSA before but did not complete the process, you can click the button and complete the account linking as described in the following steps.
+
+. Confirm that you want your contact information to be shared with Red Hat and enable the service:
++
+image::rosa-enable-2.png[]
++
+* You will not be charged by enabling the service in this step. The connection is made for billing and metering that will take place only after you deploy your first cluster.
+* This could take a few minutes:
```

Comment:
Should this be a new step? I'm a little confused on the flow to the next step.

![image](https://github.com/openshift/openshift-docs/assets/16167833/f920e44d-36fc-4852-bdd4-6666c69edb6f)

Item 3 reads like it is a continuation of this quota piece.

Reply:
The separate step (numbered as step 1.4 in the source document) should start with the text "Other sections on this verification...". The next step was at the "If all prereqs are met..." level in the original. Also, the "After the process is completed..." was marked as a separate step in the original.